docs/sdks/server-sdks/python.md

```diff
@@ -67,56 +72,62 @@ For the assignment event to send the relevant information, we have to expand the
 We have also stored out SDK Key into the environment variable `EPPO_API_KEY` as a common safety practice.
 
 ```python
-import logging, os
+import logging
+import os
 from uuid import uuid4
 from time import sleep
 import eppo_client
 from eppo_client.config import Config, AssignmentLogger
 
 logging.basicConfig(
-	filename='eppo_assignments.csv',
-	level=logging.INFO,
-	format=f'%(message)s')
+    filename='eppo_assignments.csv',
+    level=logging.INFO,
+    format=f'%(message)s')
+
 
 class LocalAssignmentLogger(AssignmentLogger):
-	def log_assignment(self, assignment):
-		logging.info(assignment)
```

Comment:
I believe there was a mix of tabs and spaces that made it so the indentation looked weird in the markdown file even though it rendered correctly in the docs

Reply:
I blame GitHub editing.